packages/flutter/lib/src/widgets/view.dart

```diff
@@ -2,48 +2,87 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'dart:ui' show FlutterView;
+import 'dart:collection';
+import 'dart:ui' show FlutterView, SemanticsUpdate;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
 
 import 'framework.dart';
 import 'lookup_boundary.dart';
 import 'media_query.dart';
 
-/// Injects a [FlutterView] into the tree and makes it available to descendants
-/// within the same [LookupBoundary] via [View.of] and [View.maybeOf].
+/// Bootstraps a render tree that is rendered into the provided [FlutterView].
+///
+/// The content rendered into that view is determined by the provided [child].
+/// Descendants within the same [LookupBoundary] can look up the view they are
+/// rendered into via [View.of] and [View.maybeOf].
 ///
 /// The provided [child] is wrapped in a [MediaQuery] constructed from the given
 /// [view].
 ///
-/// In a future version of Flutter, the functionality of this widget will be
-/// extended to actually bootstrap the render tree that is going to be rendered
-/// into the provided [view]. This will enable rendering content into multiple
-/// [FlutterView]s from a single widget tree.
-///
 /// Each [FlutterView] can be associated with at most one [View] widget in the
 /// widget tree. Two or more [View] widgets configured with the same
 /// [FlutterView] must never exist within the same widget tree at the same time.
 /// Internally, this limitation is enforced by a [GlobalObjectKey] that derives
 /// its identity from the [view] provided to this widget.
+///
+/// Since the [View] widget bootstraps its own independent render tree neither
+/// it not any of its descendants will insert a [RenderObject] into an existing
+/// render tree. Therefore, the [View] widget can only be used in those parts of
+/// the widget tree, where it is not required to participate in the construction
+/// of the surrounding render tree. In practical terms, this means it can
+/// typically be used at the root of the widget tree outside of any other [View]
+/// widget, as a child of a [ViewCollection] widget, or in the [ViewAnchor.view]
+/// slot of a [ViewAnchor] widget. It must not necessarily be a direct child,
```

Comment:
Is "must not" correct? From my understanding `View` can be a direct child if needed. Consider:

```suggestion
/// slot of a [ViewAnchor] widget. It might not be a direct child,
```